frontend/app/views/popups/eventMenu.html

```diff
@@ -0,0 +1,28 @@
+<div ng-controller="EventMenuCtrl as EventMenuCtrl" class="event-menu">
+  <div class="modal-body">
```

Comment:
if you use $scope please use follow define:
````javascript
 ng-controller="EventMenuCtrl" 
````
else use this. One semantic please.